website/addons/dataverse/settings/defaults.py

```diff
@@ -1,5 +1,6 @@
 DEFAULT_HOSTS = [
     'dataverse.harvard.edu',            # Harvard PRODUCTION server
+    'dataverse.lib.virginia.edu',       # UVA PRODUCTION server
     'dataverse-demo.iq.harvard.edu',    # Harvard DEMO server
```

Comment:
Just a heads up that we renamed http://dataverse-demo.iq.harvard.edu to https://demo.dataverse.org (it redirects so no big deal).